bundles/io/org.eclipse.smarthome.io.transport.serial/src/main/java/org/eclipse/smarthome/io/transport/serial/ProtocolType.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) 2014,2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.smarthome.io.transport.serial;
+
+import java.net.URI;
+
+/**
+ * Holds the {@link PathType}, which specifies whether its a local or remote path and the scheme.
+ *
+ * @author Matthias Steigenberger - Initial Contribution
+ *
+ */
+public class ProtocolType {
+
+    /**
+     * Remote (NET) or Local path.
+     *
+     */
+    public enum PathType {
+    NET,
```

Comment:
Are that lines correctly intended?

Reply:
imho not, but this is done by eclipse formatter automatically which I use for openhab development. Shouldn't it be used here in esh? Is it a bug in the formatter?

Reply:
I consider this as a bug in the formatter. I have seen it on other projects, too.